browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2899,6 +2905,33 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_ACCESSIBILITY_NEW_TAB_PAGE" desc="Accessibility text to read aloud when the user focuses the new tab view.">
       New tab
     </message>
+    <message name="IDS_ACCESSIBILITY_TOOLBAR_BTN_BRAVE_WALLET" desc="Content description for the button that allows an access to Brave Wallet.">
+      Access Brave Wallet
+    </message>
+    <message name="IDS_DAPP_BOTTOM_SHEET_WALLET_CONNECTIVITY_INFO" desc="Brave wallet Dapp wallet connectivity bottom sheet text">
+      Use this panel to securely access web3 and all your crypto assets.
+    </message>
+    <message name="IDS_DAPP_WALLET_PANEL_CONNECTIVITY_STATUS" desc="Brave wallet Dapp wallet connectivity status">
+      Not connected...
```

Comment:
Is the 3 dots here correct? Is it some kind of action you can click for more?

Reply:
I think so, we will need to audit all DApps strings when we are done.